cloud/docker/docker_image_facts.py

```diff
@@ -111,7 +111,7 @@
                 "Cmd": [
                     "/bin/sh",
                     "-c",
-                    "#(nop) CMD [\"/etc/docker/registry/config.yml\"]"
+                    "#(nop) CMD ['/etc/docker/registry/config.yml']"
```

Comment:
According to the upstream Dockerfile reference, this is parsed as JSON and *must* use double-quotes.

https://docs.docker.com/engine/reference/builder/#cmd